package.json

```diff
@@ -1541,10 +1541,7 @@
                 "command": "aws.submitFeedback",
                 "title": "%AWS.command.submitFeedback%",
                 "category": "%AWS.title%",
-                "icon": {
-                    "dark": "third-party/resources/from-vscode/dark/feedback.svg",
-                    "light": "third-party/resources/from-vscode/light/feedback.svg"
```

Comment:
should we delete these files also?

Reply:
Yes, they are not used elsewhere, so they can be deleted.